koku/masu/processor/parquet/parquet_report_processor.py

```diff
@@ -330,7 +330,7 @@ def prepare_parquet_s3(self, filename: os.PathLike):
         # OCP operators that send daily report files must wipe s3 before copying to prevent duplication
         if (
             not manifest_accessor.should_s3_parquet_be_cleared(manifest)
-            or manifest_accessor.get_s3_parquet_cleared(manifest, self.report_type)
+            or manifest_accessor.get_s3_parquet_cleared(manifest, filename.stem)
```

Comment:
Strip the suffixes here.

Assuming we want `pod_usage.2023-09-06.84.2_daily_0.parquet` to be pared down to `pod_usage.2023-09-06.84`. Adjust the index value as necessary.
```suggestion
            or manifest_accessor.get_s3_parquet_cleared(manifest, filename.name.removesuffix(''.join(filename.suffixes[2:]))
```